nbbuild/javadoctools/build.xml

```diff
@@ -31,7 +31,13 @@
     <import file="../default.xml"/>
 
     <target name="build-javadoc" depends="-jdk-init,-load-build-properties,init-tasks,set-buildnumber" description="Builds Javadoc documentation for modules; on branches pass e.g. -Djavadoc.web.root=http://bits.netbeans.org/6.7/javadoc">
-        <resolvelist name="allmodules" path="modules.fullpath" dir="${nb_all}" list="${nb.clusters.list}" modules="${config.javadoc.all}"/>
```

Comment:
If I understood well this change, the module list is explicit already and there's no reason to search and translate it using cluster dirs which all reside under `${nb_all}` anyway right ?

Reply:
Thanks for the review. I did a mistake by copy paste from the release90 file. I did not rethink on the modification of the cluster. The only build.xml changes are the date override.

Reply:
eh ;) I didn't look for the previous changes - apparently it was me modifying this line to work with cluster subdirs :) Bad memory...